CSS/Components/card.css

```diff
@@ -0,0 +1,87 @@
+.card-container {
+  width: 80%;
+  padding: var(--sp-md);
+  background-color: var(--gray-100);
+  box-shadow: 0 0 2px var(--gray-700);
+}
+.card {
+  width: 20rem;
```

Comment:
don't give a fixed width to the card, instead use max-width and a min-width to make it responsive and easily reusable

Reply:
Thanks for sharing, Noted and updated.